BO4E/COM/Erreichbarkeit.cs

```diff
@@ -0,0 +1,59 @@
+using System.Text.Json.Serialization;
+
+using BO4E.meta;
+
+using Newtonsoft.Json;
+
+using ProtoBuf;
+
+namespace BO4E.COM
+{
+    /// <summary>
+    /// Die Erreichbarkeit eines Unternehmens an Werktagen.
+    /// </summary>
+    [ProtoContract]
+    public class Erreichbarkeit : COM
+    {
+        /// <summary>Erreichbarkeit am Montag (Format: HHMMHHMM).</summary>
```

Comment:
Ich bin damit nicht so glücklich. Ich weiß, ws. kommt das so ausm Edifact raus, aber wir haben ja auch den Anspruch, dass wir besser lesbar sind als edifact.
Wie wäre es mit einem TimeOnly field?
https://learn.microsoft.com/en-us/dotnet/api/system.timeonly?view=net-8.0

Reply:
könnte ich es so difinnieren, aber mein die frage ist dass es ein COM object sein sollte?
```c#
public class TimeRange
    {
        public TimeOnly StartTime { get; set; }
        public TimeOnly EndTime { get; set; }

        public TimeRange(TimeOnly startTime, TimeOnly endTime)
        {
            StartTime = startTime;
            EndTime = endTime;
        }
    }

Reply:
oder `Zeitspanne`

Reply:
Oder das hier? https://github.com/Hochfrequenz/BO4E-dotnet/blob/c9038e113d5f5342c20132a040f9934e35a993dd/BO4E/COM/Zeitraum.cs#L17

oder so
https://github.com/bo4e/BO4E-python/blob/dcbb3153b188ca25b3fb2d542c45f130a8ee11e7/src/bo4e/com/zeitspanne.py#L15

oder so
https://github.com/bo4e/BO4E-python/blob/dcbb3153b188ca25b3fb2d542c45f130a8ee11e7/src/bo4e/com/zeitraum.py#L22

Reply:
also am besten kompatibel mit dem was schon da ist. calling @hf-sheese , @DeltaDaniel @hf-fvesely https://github.com/bo4e/BO4E-python/pull/625

Reply:
aber Zeitraum ein Datetime ist. Wir wollen bstimmt nicht 0000-00-00:08:00+1 setzen :D 
Ein TimeOnly basiert Zeitraum haben wir bisher in dieser Repo nicht.

Reply:
The TimeOnly and DateOnly types were introduced in .NET 6.0, which means they are not supported by any version of .NET Standard.

.NET Standard is a formal specification of .NET APIs that are intended to be available on all .NET implementations. However, .NET Standard itself has not been updated beyond .NET Standard 2.1, and TimeOnly is not available in any version of .NET Standard.

If you need to use TimeOnly, you would need to target .NET 6.0 or later, which is part of the unified .NET platform (referred to as ".NET Core" prior to .NET 5.0).

@hf-kklein sollen wir entwieder bei .net version hochsteigen oder ein `CustomTime` type diffinieren mit Stunde und Minute als Property.

Reply:
gibt ein Polyfill für TimeOnly (z.B. https://www.nuget.org/packages/Portable.System.DateTimeOnly)

Reply:
hat geklappt, danke 
https://github.com/Hochfrequenz/BO4E-dotnet/pull/466/commits/d94564f276a9f74470836ec54ec9b23488008462